pkgs/top-level/python-packages.nix

```diff
@@ -252,6 +254,8 @@ in {
 
   btchip = callPackage ../development/python-modules/btchip { };
 
+  DataModelDict = callPackage ../development/python-modules/DataModelDict { };
```

Comment:
use normalized name (lowercase and dash instead of underscore/point) for both the attribute, and the filename.

Reply:
Fixed.